assets/scss/_oss_info.scss

```
@@ -0,0 +1,138 @@
+  display: flex;
+  flex-direction: column;
+  min-width: 0;
+}
+
+.oss-card-title {
+  font-weight: 600;
+  font-size: 1rem;
+  line-height: 1.25;
+  color: inherit;
+  text-decoration: none;
+  word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)